src/archiefvernietigingscomponent/demo/management/commands/create-demo-data.py

```diff
@@ -29,14 +30,20 @@ def handle(self, *args, **options):
         assert drc_service, "No service defined for the Documenten API"
         drc_client = drc_service.build_client()
 
-        # Create a catalogus
-        catalogus = ztc_client.create(resource="catalogus", data=CATALOGUS)
+        # Create a catalogus if one doesn't exist already
+        catalogus_list = ztc_client.list(
+            resource="catalogus", query_params={"domein": "CATAL", "rsin": "104567387",}
```

Comment:
It would be best to re-use the values in the variable `CATALOGUS` defined above. Otherwise when `CATALOGUS` is changed, you would be filtering for the wrong values.